modules/compute/gke-node-pool/outputs.tf

```diff
@@ -64,3 +64,23 @@ output "tolerations" {
   description = "Tolerations needed for a pod to be scheduled on this node pool."
   value       = local.tolerations
 }
+
+output "instructions" {
+  description = "Instructions for submitting the sample GPUDirect enabled job."
+  value       = <<-EOT
+    A sample GKE job that had GPUDirect enabled and NCCL test included has been created locally at:
+      ${abspath(local.gpu_direct_setting.updated_workload_path)}
+
+    You can use the following commands to submit the sample job:
+      kubectl create -f ${abspath(local.gpu_direct_setting.updated_workload_path)}
+
+    If you would like to enable GPUDirect for your own workload, please provide the path to your Kubernetes Job manifest
+    as WORKLOAD_PATH in the command below:
```

Comment:
Can reword as follows:

Follow below steps to enable GPUDirect for your own workload:

```
export WORKLOAD_PATH=<>

python3 ${abspath("${path.module}/gpu-direct-workload/scripts/enable-tcpxo-in-workload.py")} --file $<WORKLOAD_PATH> --rxdm ${local.gpu_direct_setting.rxdm_version}
```

Or you can update your workload based on the guide at https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#add-gpudirect-manifests

Reply:
updated